src/main.tsx

```diff
@@ -20,9 +20,13 @@ Sentry.init({
   ...generateSentryConfig(),
 });
 
+// eslint-disable-next-line no-console
+console.log('MOCK_API', MOCK_API);
+
 // setup mocked api for cypress or standalone app
 /* istanbul ignore next */
 if (MOCK_API) {
+  console.info('MOCK enabled');
```

Comment:
When MSW or Mirage are active you should see their logs in the console. But it is ok to have an info log here nonetheless.